src/Native/Runtime/GCHelpers.h

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
Or maybe GCMemoryHelpers.* to make more general? It would be nice to move helpers like memcpyGCRefs into this file.